src/wallet/wallet.ts

```diff
@@ -122,18 +136,130 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     // TODO should we have a debug mode?
   }
 
+  /**
+   * Remove sensitive data from memory
+   *
+   * NOTICE: This won't remove data from memory immediately, we have to wait until javascript
+   * garbage collect it. JavaScript currently does not provide a standard way to trigger
+   * garbage collection
+   **/
+  clearSensitiveData() {
+    this.seed = null;
```

Comment:
I do agree with you, but it was done this way to maintain compatibility with the old facade that also expects the seed to be in memory on the `start` method

We could refactor this (and I think we should), but I would suggest doing this in another PR as we would have to refactor both facades and it will be a breaking change

What do you think?

Reply:
I agree. Let's create a new issue to remove this attribute from both. Thanks!

Reply:
Issue created on https://github.com/HathorNetwork/hathor-wallet-lib/issues/320